core/src/main/scala/cats/Foldable.scala

```diff
@@ -407,6 +407,31 @@ import simulacrum.typeclass
     }.toList
 
   /**
+    * Separate this Foldable into a Tuple by a separating function `A => Either[B, C]`
+    * Equivalent to `Functor#map` and then `Alternative#separate`.
```

Comment:
If we did define it in `Alternative`, we could add a consistency law checking this equality, but hey ... you can't always get what you want :smiley:

Reply:
True, I don't really mind either way tbh.
We could add a test that checks if it's consistent with List#partition though 😄